mne/time_frequency/tests/test_spectrum.py

```diff
@@ -171,6 +171,18 @@ def test_spectrum_reject_by_annot(raw):
     assert spect_no_annot != spect_reject_annot
 
 
+def test_spectrum_reject_by_exclude(raw):
+    """Test bads are not removed unless exclude="bads"."""
+    with raw.info._unlock():
+        raw.pick("mag")
+        if raw.ch_names != []:
+            raw.info["bads"] = raw.ch_names[0:1]
```

Comment:
not necessary to use `_unlock` here I don't think. `bads` is one of the fields we do allow users to change.